text/0000-export-to-oci.md

```diff
@@ -0,0 +1,192 @@
+# Meta
+[meta]: #meta
+- Name: Export to OCI format when daemon is enabled
+- Start Date: 2022-02-22
+- Author(s): Juan Bustamante (@jbustamante)
+- Status: Draft <!-- Acceptable values: Draft, Approved, On Hold, Superseded -->
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: (put "N/A" unless this replaces an existing RFC, then link to that RFC)
+
+# Summary
+[summary]: #summary
+
+When the `Exporter` phase is invoked passing the `-daemon` flag  besides writing into the Daemon also save the image to disk in [OCI Layout](https://github.com/opencontainers/image-spec/blob/main/image-layout.md) format updating the [report.toml](https://github.com/buildpacks/spec/blob/main/platform.md#reporttoml-toml) file with all the metadata require to validate the consistency of the image when it is published to a Registry.
```

Comment:
Is there any reason we can't leave this as a practical example, but add `-layout oci` as the primary interface? I don't see any reason lifecycle should restrict outputting to disk if platforms want to do that in addition to publishing to a registry.

Reply:
I think the main reason was that the daemon is the only use case where the loosing of metadata is actually an issue.  When the image is pushed to a registry, we don't have problems adding annotations or signing the image. But I agree, offering this capability for any workflow could potentially open new possibilities for Platform implementors.

Reply:
I am updating the RFC allowing to export no matter if you are exporting to a Daemon or to Registry. The idea of using a URI schema to specify different targets, it's something I will not include in this RFC because I think it could be handle later.